js/src/pages/product-feed/submission-success-guide/index.js

```diff
@@ -119,30 +203,37 @@ const pages = [
 				</cite>
 			</GuidePageContent>
 		),
+		actions: (
+			<>
+				<div className="gla-submission-success-guide__space_holder" />
```

Comment:
@ankitrox The space holder is already added here: https://github.com/woocommerce/google-listings-and-ads/pull/2943/files#diff-def2dd4fe6b3a7e9e18d76f31f0356f71bb6098dff6924a593e244c268d48a1aR146